app.legcord.Legcord.metainfo.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <!--Created with jdAppdataEdit 5.1-->
+  <id>app.legcord.Legcord</id>
+  <name>Legcord</name>
+  <summary>Legcord is a lightweight, customizable Discord client that puts you in control.</summary>
+  <developer_name>Legcord Contributors</developer_name>
+  <launchable type="desktop-id">app.legcord.Legcord.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>OSL-3.0</project_license>
+  <update_contact>support@legcord.app</update_contact>
+  <project_group>Legcord</project_group>
+  <description>
+    <p>Legcord is a lightweight, customizable Discord client that puts you in control</p>
+    <ol>
+      <li>Standalone Client - Legcord is build as a standalone client and doesn't rely on the original Discord client in any way.</li>
+      <li>Various Mods Built-in - Enjoy Vencord, Shelter, Equicord and their many features, or have a more vanilla experience, it's your choice!</li>
+      <li>Made for Privacy - Legcord automatically blocks all of Discord's trackers; even without any client mods you can feel safe and secure!</li>
+      <li>Supports Rich Presence - Unlike other clients Legcord supports rich presence (game activity) out of the box thanks to arRPC.</li>
+      <li>Mobile support - Legcord has a experimental mobile support for phones running Linux such as the PinePhone. While this is still far from ideal solution, we're slowly trying to improve it.</li>
+      <li>Much more stable - Legcord is using a newer build of Electron than the stock Discord app. This means you can have a much more stable and secure experience, along with slightly better performance.</li>
+      <li>Cross-platform support - Legcord was originally created for ARM64 Linux devices, since Discord doesn't support them. We soon decided to support every platform that Electron supports!</li>
+    </ol>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <caption>Legcord Settings</caption>
+      <image type="source">https://raw.githubusercontent.com/Legcord/Legcord/refs/heads/dev/assets/screenshots/settings.png</image>
```

Comment:
image should be referenced from an commit, not from the tip of a branch

Reply:
fixed